composer.json

```diff
@@ -39,7 +39,10 @@
         "guzzlehttp/guzzle": "^6.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.0",
-        "symfony/phpunit-bridge": "^5.0"
+        "friendsofphp/php-cs-fixer": "^3.0",
+        "symfony/phpunit-bridge": "^6.0"
```

Comment:
This dependency will require `php-7.1` version at least. And it should have the dependency issue when running `composer install` command.

Reply:
Yes, thanks for checking, I reverted back!